ga4gh/backend.py

```diff
@@ -23,14 +24,9 @@ class AbstractBackend(object):
     This class provides methods for all of the GA4GH protocol end points.
     """
     def __init__(self):
-        self._variantSetIdMap = {}
-        self._variantSetIds = []
-        self._referenceSetIdMap = {}
-        self._referenceSetIds = []
-        self._readGroupSetIdMap = {}
-        self._readGroupSetIds = []
-        self._callSetIdMap = {}
-        self._callSetIds = []
+        self._variantSetIdMap = collections.OrderedDict()
+        self._referenceSetIdMap = collections.OrderedDict()
+        self._readGroupSetIdMap = collections.OrderedDict()
```

Comment:
Before reverting any changes, it might be good to test it first and confirm that there is indeed a large slowdown.

Paul